docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -719,81 +719,80 @@ enum ReadyToRunHelper
     READYTORUN_HELPER_Unbox                     = 0x5A,
     READYTORUN_HELPER_Unbox_Nullable            = 0x5B,
     READYTORUN_HELPER_NewMultiDimArr            = 0x5C,
-    READYTORUN_HELPER_NewMultiDimArr_NonVarArg  = 0x5D,
 
     // Helpers used with generic handle lookup cases
-    READYTORUN_HELPER_NewObject                 = 0x60,
-    READYTORUN_HELPER_NewArray                  = 0x61,
-    READYTORUN_HELPER_CheckCastAny              = 0x62,
-    READYTORUN_HELPER_CheckInstanceAny          = 0x63,
-    READYTORUN_HELPER_GenericGcStaticBase       = 0x64,
-    READYTORUN_HELPER_GenericNonGcStaticBase    = 0x65,
-    READYTORUN_HELPER_GenericGcTlsBase          = 0x66,
-    READYTORUN_HELPER_GenericNonGcTlsBase       = 0x67,
-    READYTORUN_HELPER_VirtualFuncPtr            = 0x68,
+    READYTORUN_HELPER_NewObject                 = 0x5F,
```

Comment:
There are intentional holes in these ranges so that it is possible to add a new helper ID in its most logical place, without renumbering (and thus breaking) all helper ID.

Could you please revert this change?

Reply:
Also, I think we want to maintain compatibility for `ILCompiler.Reflection.ReadyToRun` (ie make it possible to use it to dump older R2R version) and re-numbering all helpers makes it hard.

Reply:
I’ll undo the renumbering and leave the hole.